exporter/redis.go

```diff
@@ -1011,15 +1016,18 @@ func (e *Exporter) scrapeRedisHost(scrapes chan<- scrapeResult, addr string, idx
 
 	if values, err := redis.Values(c.Do("SLOWLOG", "GET", "1")); err == nil {
 		var slowlogLastId int64 = 0
+		var lastSlowExecutionDurationSeconds float64 = 0
 
 		if len(values) > 0 {
 			if values, err = redis.Values(values[0], err); err == nil && len(values) > 0 {
 				slowlogLastId = values[0].(int64)
+				lastSlowExecutionDurationSeconds = float64(values[2].(int64)) / 1e6
```

Comment:
Question: line 1022 only checks for len(values) > 0 so this doesn't guarantee that we actually have three elements (you're accessing `[2]`) - has this value always been present or was it added recently and you need to make this change backward compatible?

Reply:
According to the official document of Redis, this value is available since 2.2.12. The `len(values)` in line 1022 is not less than 4 if the outer `len(values) > 0` (line 1021).

I will add the `>= 2` guard and make a new pull request.

Reply:
👍 
make it `> 2` as you want at least three elements. And then this should be ready for merge.